.papr.yml

```diff
@@ -37,20 +37,24 @@ inherit: true
 cluster:
   hosts:
     - name: testnode
-      distro: centos/7/atomic
+      distro: centos/7/atomic/continuous
   container:
       image: registry.fedoraproject.org/fedora:26
 
-context: centos/7/atomic
+context: centos/7/atomic/continuous
 
 ---
 inherit: true
 
 cluster:
   hosts:
     - name: testnode
-      distro: centos/7/atomic/continuous
+      distro: centos/7/atomic
   container:
       image: registry.fedoraproject.org/fedora:26
 
-context: centos/7/atomic/continuous
+context: centos/7/atomic
+
+tests:
+  - ./test_director '--skip-tags kernel_avc_denied'
```

Comment:
This is missing a `.`, no?

Reply:
Good catch...I was just flying through this in a fit of rage  :laughing: